style/components/stats.css

```diff
@@ -4,6 +4,7 @@
   align-items: center;
 }
 
+.stats,
```

Comment:
Dude, do not be lazy, just try placing `.stats` list somewhere outside the project details. See how it looks. 
Which side the text belongs to?
![image](https://user-images.githubusercontent.com/21981821/191443646-bb92d758-a856-4dc4-adce-b8fd79eba1b5.png)